examples/intro_tutorial.ipynb

```diff
@@ -0,0 +1,588 @@
+    "    1. Provide a concept ID, which is an ID provided in the Common Metadata Repository (CMR) metadata\n",
+    "    2. Data product short name (e.g. SENTINEL-1_INTERFEROGRAMS). \n",
+    "* `spatial`: Bounding box spatial constraints on the data. The Harmony `Bbox` class accepts spatial coordinates as decimal degrees in w, s, e, n order, where longitude = -180, 180 and latitude = -90, 90.   \n",
+    "* `temporal`: Date/time constraints on the data. The example below demonstrates temporal start and end ranges using the python `datetime` library.\n",
+    "\n",
+    "Other advanced parameters that may be of interest. Note that many reformatting or advanced projection options may not be available for your requested dataset. See the [documentation](https://harmony-py.readthedocs.io/en/latest/) for details on how to construct these parameters.\n",
+    "\n",
+    "* `crs`: Reproject the output coverage to the given CRS. Recognizes CRS types that can be inferred by gdal, including EPSG codes, Proj4 strings, and OGC URLs (http://www.opengis.net/def/crs/…)\n",
+    "* `interpolation`: specify the interpolation method used during reprojection and scaling\n",
+    "* `scale_extent`: scale the resulting coverage either among one axis to a given extent\n",
+    "* `scale_size`: scale the resulting coverage either among one axis to a given size\n",
+    "* `granule_id`: The CMR Granule ID for the granule (file) which should be retrieved\n",
+    "* `width`: number of columns to return in the output coverage\n",
+    "* `height`: number of rows to return in the output coverage\n",
+    "* `format`: the output mime type to return\n",
+    "* `max_results`: limits the number of input files processed in the request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example utilized in this tutorial demonstrates a bounding box subset over the Mauna Kea and Mauna Loa volcanoes on the island of Hawaii on February 24, 2020. The SENTINEL-1_INTERFEROGRAMS dataset, distributed by the ASF DAAC, is a prototype Level 2 NISAR-Format product. See https://asf.alaska.edu/data-sets/derived-data-sets/sentinel-1-interferograms/ for more information. \n",
+    "\n",
+    "This request specifies a subset of the unwrappedPhase variable, in TIFF format, with a maximum file result capped at 2 for demonstration purposes.\n",
+    "\n",
+    "#### ___Note that a Sentinel-3 End-User License Agreement (EULA) is required to access these data.___\n",
+    "#### ___Please visit https://urs.earthdata.nasa.gov/app_eula/sentinel and ensure that you submit this agreement form prior to executing the remainder of the notebook, otherwise you will receive an error.___"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request = Request(\n",
```

Comment:
This request doesn't work for me but I'm almost certain it's EULA-related.

Reply:
Did you manually attempt to accept the EULA through the link I included? Just curious if that worked ( I couldn't really test this myself since I had already accepted it).

Reply:
Sorry - I got interrupted in reviewing this before I saw the link (I missed your good instructions!). I had already accepted it but the request was still erroring out. It turns out the request was being made against UAT. I tried the request against PROD manually and it worked so I'm now looking into why UAT was the default.